browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -534,3 +534,7 @@ void BraveActionsContainer::ToggleExtensionsMenu() {}
 bool BraveActionsContainer::HasAnyExtensions() const {
   return false;
 }
+
+void BraveActionsContainer::UpdateToolbarActionHoverCard(
+    ToolbarActionView* action_view,
+    ToolbarActionHoverCardUpdateType update_type) {}
```

Comment:
we don't need to implement anything here? cc @petemill

Reply:
I don't think so. This is a new feature behind a flag and we haven't talked about what to do with it for Shields or Rewards yet. https://chromium.googlesource.com/chromium/src/+/c313724030f1fa5dcf1e2ca1ce127b00e1497aa2